assets/src/dashboard/components/cardGridItem/cardPreview.js

```diff
@@ -99,7 +99,12 @@ const EmptyActionContainer = styled(ActionContainer)`
 
 const getActionAttributes = (targetAction) =>
   typeof targetAction === 'string'
-    ? { href: resolveRoute(targetAction), isLink: true }
+    ? {
+        href: resolveRoute(targetAction),
+        isLink: true,
+        target: '_blank',
```

Comment:
Why open in a new tab?

Shouldn't we let this up to the user to decide? That's the common a11y practice here AFAIK.

New tab might make sense for external links, but for the editor less so.

Reply:
I was following the ticket that was created.

<img width="713" alt="Screen Shot 2020-09-17 at 9 49 19 AM" src="https://user-images.githubusercontent.com/1738349/93487613-130ead80-f8cb-11ea-9299-1c317e01db73.png">

Reply:
I don't mind dropping it if we want to change the expected behavior.

Reply:
I guess to be clear what do we want?

1. Neither the "Open in Editor" context menu or the "Open in Editor" button on the card to open in a new tab
2. Only the Context menu item to open in a new tab
3. Both of them to open in a new tab

Reply:
Hmm looks my ticket description was unclear, sorry about that!

The key aspect is that these become links, not buttons. Then they will be announced as such to screen readers, and I can hover over them to see the target URL, click on them, or right-click and open in a new tab. The latter was just an example I was using on the ticket, not the desired default behavior.

Opening links in new tabs by default is not good for accessibility. Citing WCAG recommendations:

> In general, it is better not to open new windows and tabs since they can be disorienting for people, especially people who have difficulty perceiving visual content.

> Neither the "Open in Editor" context menu or the "Open in Editor" button on the card to open in a new tab

This should be the behavior :-)

Reply:
Oh no worries! I do get what you mean now because since they are `a` tags they can be right clicked to launch a new tab. Sorry about that. I will update to regular behavior.